src/plans/quasi_newton_plan.jl

```diff
@@ -467,21 +465,50 @@ function QuasiNewtonLimitedMemoryDirectionUpdate(
         convert(mT, scale),
         project,
         vector_transport_method,
+        "",
     )
 end
+get_message(d::QuasiNewtonLimitedMemoryDirectionUpdate) = d.message
+function status_summary(d::QuasiNewtonLimitedMemoryDirectionUpdate{T}) where {T}
+    s = "limited memory $T (size $(length(d.memory_s)))"
+    (d.scale != 1.0) && (s = "$(s) initial scaling $(d.scale)")
+    d.project && (s = "$(s), projections, ")
+    s = "$(s)and $(d.vector_transport_method) as vector transport."
+    return s
+end
 function (d::QuasiNewtonLimitedMemoryDirectionUpdate{InverseBFGS})(mp, st)
+    (length(d.message) > 0) && (d.message = "") # reset message
```

Comment:
```suggestion
    isempty(d.message) || (d.message = "") # reset message
```
`isempty` might be a bit nicer than computing length.